python/ray/util/spark/cluster_init.py

```diff
@@ -935,38 +952,76 @@ def setup_ray_cluster(
         else:
             support_stage_scheduling = False
 
+    if "num_cpus_per_node" in kwargs:
+        if num_cpus_worker_node is not None:
+            raise ValueError(
+                "'num_cpus_per_node' and 'num_cpus_worker_node' arguments cannot be "
+                "set together."
```

Comment:
`num_cpus_per_node` and `num_cpus_worker_node` arguments are equivalent. Only set `num_cpus_worker_node`.